dotcms-integration/src/test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java

```diff
@@ -1958,4 +1962,180 @@ public void Test_Rendering_Working_Content_Using_Limited_User() throws Exception
 
     }
 
+    /**
+     * Test for GitHub Issue #32261: Content with invalid date configuration should be filtered in LIVE mode.
+     */
+    @Test
```

Comment:
We try to doc our test in such a way that we describe the scenario and the expected results as follows:
Give that a contentlet with invalid dates rage set up yadda yadda yadda 
Expected: The Page Resource should render contents with a valid date range and skip invalid yadda yadda yadaa